builtin/logical/transit/path_sign_verify.go

```diff
@@ -131,6 +131,12 @@ Options are 'pss' or 'pkcs1v15'. Defaults to 'pss'`,
 				Default:     "asn1",
 				Description: `The method by which to marshal the signature. The default is 'asn1' which is used by openssl and X.509. It can also be set to 'jws' which is used for JWT signatures; setting it to this will also cause the encoding of the signature to be url-safe base64 instead of using standard base64 encoding. Currently only valid for ECDSA P-256 key types".`,
 			},
+
+			"salt_length": {
+				Type: framework.TypeString,
```

Comment:
Could we specify `Default: "auto",` within the two field definitions? It helps document things a bit more and with some code gen efforts.